src/Columns/IColumn.h

```diff
@@ -225,6 +225,12 @@ class IColumn : public COW<IColumn>
     using Filter = PaddedPODArray<UInt8>;
     virtual Ptr filter(const Filter & filt, ssize_t result_size_hint) const = 0;
 
+    /// Replace elements that match the filter with defaults. If inverted replaces not matched elements.
+    virtual Ptr setDefaults(const Filter &, bool /*inverted*/) const
```

Comment:
This method is not used. Can we remove it?

Reply:
Look at `ColumnNullable::getNestedColumnCopy()`. We need zero UInt8 values if we get nested column from nullable column in cases where NULL equivalent to zero. In this PR it called from `if` over Nullable. There we change conditional column with nested one.

Reply:
Ok.
Probably, we can use logical `and` function with nested column and null mask?
I just don't like the fact that this function is not implemented for most of the columns. Also it doesn't speed up anything.